docker.gradle

```diff
@@ -19,7 +19,7 @@
 ext {
     dockerRegistry = project.hasProperty('dockerRegistry') ? dockerRegistry + '/' : ''
     dockerImageTag = project.hasProperty('dockerImageTag') ? dockerImageTag : 'latest'
-    dockerTimeout = project.hasProperty('dockerTimeout') ? dockerTimeout.toInteger() : 420
+    dockerTimeout = project.hasProperty('dockerTimeout') ? dockerTimeout.toInteger() : 480
```

Comment:
Why specific increasing the timeout by 1 minute? should we make it larger I think I heard @rabbah saying in some cases 800 is good value.